### ISSUE TITLE
Investigate/fix error printing

### ISSUE THREAD

**Issue by Jand42** [dotnet-websharper/core#1161]
When doing a build, WebSharper errors do not show up in Visual Studio (tested with 16.10.2 and 17.0.0 Preview 1)

Investigate, ideally both C#/F# WebSharper errors during build should be picked up during build and displayed in both VS/VSCode as red squiggles, maybe WebSharper needs to make changes to match expected printing format again

**Comment by mrtank** [comment]
For a minimal
```
[<JavaScript>]
module Client =
    [<Inline "console.log('asd)">]
    let test () = X<unit>

    let Main () =
        test()
        Doc.Empty
```
Client the build fails as expected with `WS9001` error. But the squiggle is still missing (VS 2019 Community (non-preview))
![image](https://user-images.githubusercontent.com/918230/132719041-b44b1e8d-f742-4036-a793-f00dccece660.png)

**Comment by Jand42** [comment]
Tested on WS6+VS2022, F# error reporting works.
C# error reporting relies on the analyzer so that errors are not double-reported. For fixes on analyzer see #1243.

**Comment by Jand42** [comment]
WebSharper error reporting works only partially on on WS6+VS2022. Error is reported, however, we have no code service for F# with WebSharper errors, so error squiggly is hidden by F# code service not reporting the same. I don't see any current workaround other than Roslyn-analyzer like support for F#, which currently only exists for ionide https://github.com/ionide/FSharp.Analyzers.SDK.
